### PR TITLE
Provide completion for referred keywords required using :as-alias

### PR DESCRIPTION
**Pull request by rrudakov**
Before submitting a PR make sure the following things have been done:

- [x] The commits are consistent with our [contribution guidelines](../blob/master/.github/CONTRIBUTING.md)
- [x] You've added tests to cover your change(s)
- [x] All tests are passing
- [x] The new code is not generating reflection warnings
- [x] You've updated the [changelog](../blob/master/CHANGELOG.md) (if adding/changing user-visible functionality)

Thanks!

### CODE REVIEW

File: src/main/suitable/compliment/sources/cljs/analysis.clj [hunk 1]
```diff
@@ -66,10 +66,11 @@
   "Returns a map {ns-name-or-alias ns-name} for the given namespace."
   [env ns]
   (when-let [found (find-ns env ns)]
-    (let [imports (:imports found)]
+    (let [imports    (:imports found)
+          as-aliases (get found :as-aliases {})]
       (->> (:requires found)
            (filter #(not (contains? imports (key %))))
-           (into {})))))
+           (into as-aliases)))))
```
Comment (vemv) [inline src/main/suitable/compliment/sources/cljs/analysis.clj:73]: We could use transfucer form at this point

**Review by vemv — APPROVED** [review]
lgtm - thanks!

File: src/main/suitable/compliment/sources/cljs/analysis.clj (hunk 1 above)
Comment (vemv) [reply src/main/suitable/compliment/sources/cljs/analysis.clj:73]: (while we're here filter+not -> remove)

Comment (rrudakov) [reply src/main/suitable/compliment/sources/cljs/analysis.clj]: Good idea. Done!

**Comment by vemv** [comment]
🍻!

Shall we cut a release, or you having something else on the radar?

**Comment by rrudakov** [comment]
Let's cut a release. I've been looking into another issue you've mentioned earlier, but it's a bit tricky and will require a bit more discussion.

**Comment by vemv** [comment]
Suitable has been cut!

cider-nrepl not (should be later in the week), but I bumped Suitable there, so you can in the meantime perform a [local install](https://github.com/clojure-emacs/cider-nrepl#local-development)

**Comment by rrudakov** [comment]
Thank you! :beers:

**Pull request merged**